### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by quinnha**
Updated Readme with relevant information

### CODE REVIEW

**Comment by quinnha** [comment]
Add comments for whatever else you want in here @wastadtlander, esp for goals

Note: we should have a icon/picture/logo thats cute because i like cute things, and we should have a vid with the demo in eventually

File: README.md [hunk 1]
```diff
@@ -1,9 +1,50 @@
...
-2. just clone the repo 
-3. go to `chrome://extensions/`
-4. top right turn dev mode on
-5. load unpacked top left
-6. select the folder the repo is in
-7. then if u have an api key it will work when u click on any comment
\ No newline at end of file
+# Youtube Comment Translator
+
+A Chrome extension to uutomatically translate Youtube comments
```
Comment (wastadtlander) [inline README.md]: Spelling moment.

**Review by wastadtlander — APPROVED** [review]
I can't add because u werent a collab before this so no branch but can u add in the todo, eventually package extension and launch to store.

File: README.md (hunk 1 above)
Comment (quinnha) [reply README.md]: man

**Comment by quinnha** [comment]
give it a sweet lgtm and lets get this money

**Pull request merged**